### DISCUSSION TITLE
Folder of item

### DISCUSSION THREAD

**Discussion by LupusArctos4** [shedaniel/RoughlyEnoughItems#1491] [Q & A] [answered]
in a server, i have lots of custom items and i want create folder for some types of items, is it possible?

**Comment by shedaniel** [comment, marked as answer]
It is only client side, you can't do it on the server. And for that you'll want KubeJS

**Reply by LupusArctos4** [reply]
I don't think I've made it clear what I want.
I want to make a folder to organize items by type.
